search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/ext/GenerativeQAParameters.java

```diff
@@ -24,13 +24,15 @@
 import java.util.List;
 import java.util.Objects;
 
+import org.opensearch.Version;
```

Comment:
This import is useless I believe?

Reply:
Because we migrate to `CommonValue.VERSION_2_18_0;` remember? I think it's because I forgot to remove it LOL

Reply:
Wait...you are right. I found where we use it.